python/katana_setup.py

```diff
@@ -33,6 +29,111 @@ def unique_list(l):
     return list(dict.fromkeys(l))
 
 
+class RequirementError(RuntimeError):
+    def __str__(self):
+        return (
+            super(RequirementError, self).__str__()
+            + " (Normal Katana builds should use cmake to start the build and NOT directly call setup.py)"
+        )
+
+
+@lru_cache(maxsize=None)
+def require_python_module(module_name, ge_version=None, lt_version=None):
+    v_str = ""
+    if ge_version:
+        v_str += f">={ge_version}"
+    if lt_version:
+        v_str += f"<{lt_version}"
+    if ge_version or lt_version:
+        v_str = f" ({v_str})"
+    print(f"Checking for Python package '{module_name}'{v_str}...", end="")
```

Comment:
qq: why the single quotes around `module_name` here but not in the exception messages below?